src/component/marker/markerHelper.ts

```diff
@@ -147,6 +147,9 @@ export function dataTransform(
             item.coord = coord;
         }
     }
+    if (item.coord == null) {
+        item.coord = [];
```

Comment:
I'm not sure but returning an array with 0 element doesn't seem to be *valid* by logic for the `coord`. What do you think if it returns `[null, null]`? Can you help test if it works?

Reply:
I think it's fine to return an empty array. But I found another potential bug when `coord` includes the special type string like [`min/max/median/average`](https://echarts.apache.org/zh/option.html#series-bar.markArea.data:~:text=%7D%0A%20%20%20%20%5D%2C%20%5B%0A%20%20%20%20%20%20%20%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20name%3A%20%27-,%E6%89%80%E6%9C%89%E6%95%B0%E6%8D%AE%E8%8C%83%E5%9B%B4%E5%8C%BA%E9%97%B4,-%27%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20coord%3A%20%5B). Just fixed it and added a test case.